exercises/two-bucket/src/test/java/TwoBucketTest.java

```diff
@@ -0,0 +1,85 @@
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.experimental.runners.Enclosed;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class TwoBucketTest {
+	
+	@Test
+	public void testBucketOneSizeThreeBucketTwoSizeFiveStartWithOne() {
+		
+		TwoBucket twoBucket = new TwoBucket(3, 5, 1, "one");
+		
+		assertEquals(twoBucket.getTotalMoves(), 4);
```

Comment:
[`assertEquals`](http://junit.sourceforge.net/javadoc/org/junit/Assert.html) takes expected as the first argument and actual as the second so this (and all following assertions) should be:
```
assertEquals(4, twoBucket.getTotalMoves());
```
Otherwise the error message will be confusing 🙂